src/TrinaryLogic.php

```diff
@@ -92,6 +92,9 @@ public function or(self ...$operands): self
 
 	public static function extremeIdentity(self ...$operands): self
 	{
+		if ($operands === []) {
+			throw new ShouldNotHappenException();
+		}
 		$operandValues = array_column($operands, 'value');
```

Comment:
What's the current behaviour with an empty array? If the code doesn't crash then this is a BC break.

Reply:
`min([])` and `max([])` cause fatal errors on PHP >= 8. On PHP < 8 it causes a warning and returns `false`, which gets coerced to `0` when given to `create()`. See https://3v4l.org/RS767